kolibri_explore_plugin/views.py

```diff
@@ -92,3 +98,53 @@ def get(self, request, app):
         filename = self._get_file(app, "metadata.json")
         with open(filename) as json_file:
             return HttpResponse(json_file, content_type="application/json")
+
+
+@method_decorator(csrf_exempt, name="dispatch")
+class EndlessLearningCollection(View):
+    COLLECTION_TOKEN = "totoj-jupak"
+    BASE_URL = "https://kolibri-content.endlessos.org/"
+
+    def get(self, request):
+        job_ids = request.session.get("job_ids", [])
+        jobs_response = [
+            _job_to_response(queue.fetch_job(job)) for job in job_ids
+        ]
+
+        return HttpResponse(
+            json.dumps(jobs_response), content_type="application/json"
+        )
+
+    def post(self, request):
+        token = self.COLLECTION_TOKEN
+
+        channel_viewset = RemoteChannelViewSet()
+        channels = channel_viewset._make_channel_endpoint_request(
+            identifier=token
+        )
+
+        job_ids = []
+
+        for c in channels:
+            task = {
+                "channel_id": c["id"],
+                "channel_name": c["name"],
+                "baseurl": self.BASE_URL,
+                "started_by_username": "endless",
+                "type": "REMOTECHANNELIMPORT",
+            }
```

Comment:
Let's replace the single-letter variable name here :)

```suggestion
        for channel in channels:
            task = {
                "channel_id": channel["id"],
                "channel_name": channel["name"],
                "baseurl": self.BASE_URL,
                "started_by_username": "endless",
                "type": "REMOTECHANNELIMPORT",
            }
```